### ISSUE TITLE
relatedModels listener does not work for Crud.view method

### ISSUE THREAD

**Issue by phishy** [FriendsOfCake/crud#494]
Is there a reason this isn't supported out of the box? I easily added a beforeFind method to RelatedModelsListener and registered the event and it works fine. Just curious before I submit a pull request.

**Comment by josegonzalez** [comment]
None that I can think of.

**Comment by josegonzalez** [comment]
https://github.com/FriendsOfCake/crud/pull/448

**Comment by phishy** [comment]
Yeah, seems about right.

**Comment by ayman-alkom** [comment]
+1

**Comment by viniciusbig** [comment]
So, I saw #448 and #505 and I couldn't understand this.

I'm using Crud plugin without CrudView.

I've loaded the listeners and actions in my AppController to make it global.

My Clients model belongsTo an Account

So, using a regular view controller, I can get the account like this:
`$client = $this->Clients->get($id, [ 'contain' => ['Accounts'] ]);`

but using the default View action from CRUD, I don't have this association.

Even loading the relatedModels in the view action

```
$this->loadComponent('Crud.Crud', [
    'actions' => [
        'view' => [
            'className' => 'Crud.View',
            'relatedModels' => true
        ],
    ],
    'listeners' => [
        'relatedModels',
        'Crud.RelatedModels',
    ]
]);
```

How can I use the view action and still get the associated data?
